### ISSUE TITLE
Add ordinal/car name

### ISSUE THREAD

**Issue by Ldalvik** [austinbaccus/forza-telemetry#55]
Not so much a bug but a suggestion.

Here's the ordinal to car database if you want to add the name of the car you are driving to your app. If you want to message me on discord I can send you a more updated CSV, but this is the most recent one we have uploaded if you want to experiment with it for now.

https://github.com/Ldalvik/optn.club/blob/main/data/CarClasses.csv

**Comment by austinbaccus** [comment]
@Ldalvik is this data universal between FM7/FH5?

**Comment by Ldalvik** [comment]
Yes, it's universal across all the Forza releases. This one is specifically extracted from FH5, if you wanted ordinal lists for older games I'm sure you could find something online

**Comment by austinbaccus** [comment]
@Ldalvik awesome stuff. I'll definitely look into this when I get the time.
